dipy/direction/probabilistic_direction_getter.pyx

```diff
@@ -15,6 +15,7 @@ cimport numpy as cnp
 from dipy.direction.closest_peak_direction_getter cimport PmfGenDirectionGetter
 from dipy.direction.peaks import peak_directions, default_sphere
 from dipy.direction.pmf cimport PmfGen, SimplePmfGen, SHCoeffPmfGen
+from dipy.tracking.stopping_criterion cimport StoppingCriterion
```

Comment:
I do not think this import is needed. This is not used

Reply:
yes, removed.